### DISCUSSION TITLE
Mods vs ollama

### DISCUSSION THREAD

**Discussion by godalming123** [charmbracelet/mods#170] [General]
Hi, https://ollama.ai seems to be offering a very similar product. From my understanding, mods allows for context input via the command line and the use of open AI, whereas ollama allows for much easier use without complex setup. Are their any other diferences?

**Comment by sheldonhull** [comment]
mods is a cli to automate calling input/output.
Ollama is a local runner for ai models.

Think of this like a github cli automates calling github releases, pull requests and such, but isn't really the repo. Maybe that would help?
